<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create list of happy birthdays</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>256</width>
      <height>291</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>280</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Путь</string>
    </property>
    <widget class="QLineEdit" name="line_path">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Путь</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clear_path">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Очистить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_path_dir">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_FAQ_path">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Ввод даты рождения</string>
    </property>
    <widget class="QLineEdit" name="line_date">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>02.12</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_FAQ_date">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_load_data">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Загрузить записи</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_info_names_in_date">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clear_date">
     <property name="geometry">
      <rect>
       <x>145</x>
       <y>60</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Очистить</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>280</y>
      <width>341</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Добавление записей</string>
    </property>
    <widget class="QLineEdit" name="line_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Имя</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_FAQ_name">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clear_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Очистить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_add_in_list">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить к списку</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_add_in_dict">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>70</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранить список</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_exit">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>400</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Закрыть</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_clear_path</sender>
   <signal>clicked()</signal>
   <receiver>line_path</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_clear_name</sender>
   <signal>clicked()</signal>
   <receiver>line_name</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_clear_date</sender>
   <signal>clicked()</signal>
   <receiver>line_date</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
